cmd/internal/cli/singularity.go

```diff
@@ -243,25 +249,30 @@ func ExecuteSingularity() {
 		sylog.Fatalf("CLI command manager reported %d error(s)", cliErrors)
 	}
 
-	if cmd, err := SingularityCmd.ExecuteC(); err != nil {
+	if cmd, err := singularityCmd.ExecuteC(); err != nil {
 		name := cmd.Name()
 		switch err.(type) {
 		case cmdline.FlagError:
 			usage := cmd.Flags().FlagUsagesWrapped(getColumns())
-			SingularityCmd.Printf("Error for command %q: %s\n\n", name, err)
-			SingularityCmd.Printf("Options for %s command:\n\n%s\n", name, usage)
+			singularityCmd.Printf("Error for command %q: %s\n\n", name, err)
+			singularityCmd.Printf("Options for %s command:\n\n%s\n", name, usage)
 		case cmdline.CommandError:
-			SingularityCmd.Println(cmd.UsageString())
+			singularityCmd.Println(cmd.UsageString())
 		default:
-			SingularityCmd.Printf("Error for command %q: %s\n\n", name, err)
-			SingularityCmd.Println(cmd.UsageString())
+			singularityCmd.Printf("Error for command %q: %s\n\n", name, err)
+			singularityCmd.Println(cmd.UsageString())
 		}
-		SingularityCmd.Printf("Run '%s --help' for more detailed usage information.\n",
+		singularityCmd.Printf("Run '%s --help' for more detailed usage information.\n",
 			cmd.CommandPath())
 		os.Exit(1)
 	}
 }
 
+// GenBashCompletionFile
+func GenBashCompletion(w io.Writer) error {
+	return singularityCmd.GenBashCompletion(fn)
```

Comment:
Looks like this should be passing `w` instead of `fn`

Reply:
LOL, thanks. I edited something at the last minute and forgot about it.

Reply:
```suggestion
	return singularityCmd.GenBashCompletion(w)
```